gtsam/linear/VectorValues.cpp

```diff
@@ -92,7 +92,7 @@ namespace gtsam {
       // Use this trick to find the value using a hint, since we are inserting
       // from another sorted map
       size_t oldSize = values_.size();
-      hint = values_.emplace_hint(hint, key, value);
+      hint = values_.insert(hint, {key, value});
```

Comment:
Is this related to the PR or just random?

Reply:
It fails in develop !!! It snuck in so I put the fix in here to save on CI..